extensions/amp-story/1.0/amp-story-reaction-quiz.js

```diff
@@ -103,8 +105,8 @@ export class AmpStoryReactionQuiz extends AmpStoryReaction {
 
     // First child must be heading h1-h3
     if (!['h1', 'h2', 'h3'].includes(promptInput.tagName.toLowerCase())) {
-      dev().error(
-        TAG,
+      devAssert(
+        false,
```

Comment:
dev().error() didn't get caught on the tests so the tests weren't working at all, devAssert does throw the error so tests work now.